.circleci/config.yml

```diff
@@ -257,6 +273,9 @@ workflows:
       - build-icmp-responder-nse:
           requires:
             - sanity-check
+      - build-vppagent-icmp-responder-nse:
```

Comment:
We should make the integration tests depend on this task.

Reply:
Fixed